docs/usage/services.rst

```diff
@@ -89,37 +89,39 @@ Services provide high-level methods for common operations:
             data=data,
             auto_commit=True,
         )
-        return post_service.to_schema(post)
+        return post_service.to_schema(post, schema_type=PostResponse)
 
 Complex Operations
 -------------------
 
-Services can handle complex business logic involving multiple repositories:
```

Comment:
afaict, the example code doesn't actually use multiple repositories. i think it'd be good to eventually have an example of multiple repos, but that can happen later

Reply:
This is also correct.  I guess I forgot to come back to this one.